docs/usage/visualize.ipynb

```diff
@@ -539,6 +533,66 @@
     "graphviz.Source(dot)"
```

Comment:
(line above) I think it would be cool here to also use `collapse_graphs=True`, now that your code should be able to handle that. You can also remove the `[::10]` selector then.